be/src/storage/persistent_index.cpp

```diff
@@ -488,14 +496,68 @@ Status ImmutableIndexWriter::write_shard(size_t key_size, size_t npage_hint, siz
         }
         _cur_key_size = key_size;
     }
+#ifdef BE_TEST
+    if (config::enable_pindex_filter) {
```

Comment:
Why check BE_TEST?

Reply:
> Why check BE_TEST?

I want to check the correctness and compatibility of bloom filter. So I need to construct some rowset file without blomm filter. So  I use `enable_pindex_filter` to decide if I need to generate a bloom filter when writing persistent index file.

Reply:
If so, looks like `enable_pindex_filter` is used with different semantic, a better way is to add a control global variable like `write_pindex_bf` and
```
bool write_pindex_bf = true;
if (write_pindex_bf) {
 // work
}

// test.cpp
extern bool write_pindex_bf;
write_pindex_bf = false;
```